src/components/repository/IntentLabelCard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <bh-card>
+    <div class="bh-card__left__content__flex-display">
+      <strong> {{ title }}</strong>
+      <div class="bh-card__left__content__flex-display__label">
+        <p><strong>{{ 0 }}</strong> Sentences </p>
+        <p v-if="entities.length != 0 "><strong>{{ 0 }}</strong> Entities</p>
+      </div>
+    </div>
+    <p v-if="entities.length != 0">Entities</p>
+    <template slot="right">
+      <bh-badge
+        v-for="(entity, i) in entities"
+        :key="i"
+        :class="[
+          getEntityClass(entity),
+          'bh-card__right__space-around'
```

Comment:
Don't use classes from another components, maybe this can be fix with grid system.